include/private/rdycoreimpl.h

```diff
@@ -37,6 +37,9 @@ typedef struct {
       PetscReal water_mass;
       PetscReal x_momentum;
       PetscReal y_momentum;
+      PetscReal water_mass_prev;
+      PetscReal x_momentum_prev;
+      PetscReal y_momentum_prev;
```

Comment:
An alternative to adding `prev` fields to this struct is to

* Define a `TimeSeriesBoundaryFlux` struct like this (instead of using an anonymous struct):
```
typedef struct {
struct {
  PetscReal water_mass;
  PetscReal x_momentum;
  PetscReal y_momentum;
} TimeSeriesBoundaryFlux;
```
* Replace the `fluxes` field in `boundary_fluxes` with the following:
```
  TimeSeriesBoundaryFlux current, previous;
```